src/katgpucbf/send.py

```diff
@@ -53,10 +56,13 @@ def __init__(
         descriptors: spead2.send.Heap,
         interval: float,
         first_interval: Optional[float] = None,
+        send_all_descriptors: Optional[bool] = False,
```

Comment:
```suggestion
        send_all_descriptors: bool = False,
```
Otherwise you're saying that it can be `None` as well, and I don't see a need for that.

Reply:
I'd suggest making this a keyword-only argument, since you're already calling it as a keyword argument (which I like for boolean flags since it makes the purpose of the argument clear, compared to just seeing an argument that is `True`).

Reply:
This is a fair point, I concur.